lib/rules/value-no-vendor-prefix/index.mjs

```diff
@@ -52,8 +53,23 @@ const rule = (primary, secondaryOptions, context) => {
 				return;
 			}
 
-			if (optionsMatches(secondaryOptions, 'ignoreValues', vendor.unprefixed(value))) {
-				return;
+			if (secondaryOptions?.ignoreValues) {
+				const isIgnored = secondaryOptions.ignoreValues.some(
+					(/** @type { string | RegExp } */ element) => {
+						const isStringifiedRegex =
+							isString(element) && element[0] === '/' && element.at(-1) === '/';
+						const useRawValue = element instanceof RegExp || isStringifiedRegex;
```

Comment:
[suggest]

- `isRegExp()` is available also here
- `startsWith()` and `endsWith()` can make it a bit readable, IMO
- Lazily evaluating a string `element` (e.g. `isString()` etc.) seems a bit more performant

```suggestion
						const useRawValue = isRegExp(element) || (isString(element) && element.startsWith('/') && element.endsWith('/'));
```